packages/kbn-reporting/export_types_helpers_public/report_types.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+// Export Type Definitions
+export const CSV_REPORT_TYPE = 'csv_searchsource';
```

Comment:
This is exported from a few places. Search the code for `export const .*_REPORT_TYPE`